### PR TITLE
feat: INFRA-454 Added docker regitry auth

### PR DESCRIPTION
**Pull request by ihannache**

### CODE REVIEW

**Review by sebastienbonami — APPROVED** [review]

**Comment by Magnitus-** [comment]
@ihannache You will also need to add the adjustment so that the os can be passed via an existing volume

**Comment by Magnitus-** [comment]
I'd follow the same format as our other modules for input:
https://github.com/Ferlab-Ste-Justine/terraform-openstack-coredns-server/blob/main/variables.tf#L7
https://github.com/Ferlab-Ste-Justine/terraform-openstack-etcd-server/blob/main/variables.tf#L7
https://github.com/Ferlab-Ste-Justine/openstack-postgres-standalone/blob/master/variable.tf#L7
https://github.com/Ferlab-Ste-Justine/terraform-openstack-opensearch-server/blob/main/variables.tf#L16
...

Its describes more clearly what is expected which is either that:
- We are passing the id of an image directly
- We are passing the id of a volume based on the image

File: README.md [hunk 1]
```diff
@@ -12,14 +12,21 @@ This terraform module provisions a base vm that can be setup as a worker or mast
 - **image_id**: Id of the vm image used to provision the node
 - **flavor_id**: Id of the VM flavor
 - **keypair_name**: Name of the keypair that will be used to ssh to the node
+- **boot_from_volume**: Whether to boot the instance from a volume
+- **volume_id**: Id of the disk volume to attach to the vm
```
Comment (Magnitus-) [inline README.md]: This needs to be updated.

**Review by Magnitus- — APPROVED** [review]
LGTM

**Pull request merged**